Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "325a3b81f83f06bc671c6af64fe576536ad4a8eea219dee603136d58feab5a78"
```

Comment:
I am not sure right now, if it is possible with pipenv, but if, it would be better to just have the changes of the install in here without any other updates.

Reply:
I tried and failed. It seems to automatically update all dependecies that are not pinned to a specific version as soon as I run `pipenv update` or `pipenv lock`

Reply:
Pity, it behaves like that. Alright  then.